lovefield/lovefield.d.ts

```diff
@@ -158,12 +158,14 @@ declare namespace lf {
       getNormalizedName(): string
     }
 
-    export interface Table {
+    interface ITable {
```

Comment:
Can you explain what error is this fixing? There are several references to `schema.Table` throughout this file, and no reference to `ITable`.